components/Dialogs/src/MDCDialogTransitionController.m

```diff
@@ -52,6 +72,19 @@ - (void)animateTransition:(id<UIViewControllerContextTransitioning>)transitionCo
 
   UIView *containerView = transitionContext.containerView;
 
+  [CATransaction setCompletionBlock:^{
+    // If we're dismissing, remove the presented view from the hierarchy
+    if (!presenting) {
+      [fromView removeFromSuperview];
+    }
+
+    // From ADC : UIViewControllerContextTransitioning
+    // When you do create transition animations, always call the
+    // completeTransition: from an appropriate completion block to let UIKit know
```

Comment:
Please create a link to the source of this comment. I'm not sure what "ADC" means.

Reply:
I'm not sure either (this comment existed before I started work on this PR), but I added a link to what I think is the relevant documentation.